js/data.js

```diff
@@ -0,0 +1,75 @@
+import { getRandomNumber } from './util.js';
+
+export function generateRandomComment() {
+  const avatars = [
+    'img/avatar-1.svg',
```

Comment:
Delete this constant because you have many same values. Think about use this img paths diference.